.github/workflows/submodule-pin-check.yml

```diff
@@ -0,0 +1,20 @@
+name: Merge Checks
```

Comment:
```suggestion
name: Submodule Pin Check
```

Reply:
That was just a cut and paste error, thanks for spotting it.